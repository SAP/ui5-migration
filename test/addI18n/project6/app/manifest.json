{
	"_version": "1.1.0",

	"sap.app": {
		"_version": "1.1.0",
		"id": "myapp.app",
		"type": "application",
		"i18n": "../messagebundle.properties",
		"applicationVersion": {
			"version": "${project.version}"
		},
		"title": "{{TITLE}}",
		"description": "{{DESCRIPTION}}",
		"tags": {
			"keywords": ["{{TITLE}}"]
		},
		"ach": "ME",
		"offline": false,
		"resources": "resources.json"
	},

	"sap.ui": {
		"_version": "1.1.0",
		"technology": "UI5",
		"deviceTypes": {
			"desktop": true,
			"tablet": true,
			"phone": true
		}
	},

	"sap.ui5": {
		"_version": "1.1.0",
		"dependencies": {
			"minUI5Version": "1.30.1",
			"libs": {
				"sap.ui.core": {
					"minVersion": "1.30.1"
				},
				"sap.ui.comp": {
					"minVersion": "1.30.1"
				}
			}
		},

		"models": {
			"i18n": {
				"type": "sap.ui.model.resource.ResourceModel",
				"uri": "../messagebundle.properties"
			}
		},

		"contentDensities": {
			"compact": true,
			"cozy": true
		}
	}
}

