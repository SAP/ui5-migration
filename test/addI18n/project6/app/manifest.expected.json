{
	"_version": "1.21.0",

	"sap.app": {
		"_version": "1.1.0",
		"id": "myapp.app",
		"type": "application",
		"i18n": {
			"bundleUrl": "../messagebundle.properties",
			"supportedLocales": [
				"",
				"ar",
				"sh"
			],
			"fallbackLocale": ""
		},
		"applicationVersion": {
			"version": "${project.version}"
		},
		"title": "{{TITLE}}",
		"description": "{{DESCRIPTION}}",
		"tags": {
			"keywords": ["{{TITLE}}"]
		},
		"ach": "ME",
		"offline": false,
		"resources": "resources.json"
	},

	"sap.ui": {
		"_version": "1.1.0",
		"technology": "UI5",
		"deviceTypes": {
			"desktop": true,
			"tablet": true,
			"phone": true
		}
	},

	"sap.ui5": {
		"_version": "1.1.0",
		"dependencies": {
			"minUI5Version": "1.30.1",
			"libs": {
				"sap.ui.core": {
					"minVersion": "1.30.1"
				},
				"sap.ui.comp": {
					"minVersion": "1.30.1"
				}
			}
		},

		"models": {
			"i18n": {
				"type": "sap.ui.model.resource.ResourceModel",
				"uri": "../messagebundle.properties",
				"settings": {
					"supportedLocales": [
						"",
						"ar",
						"sh"
					],
					"fallbackLocale": ""
				}
			}
		},

		"contentDensities": {
			"compact": true,
			"cozy": true
		}
	}
}

